### ISSUE TITLE
Statistics in dashboard don't work

### ISSUE THREAD

**Issue by santo74** [arangodb/arangodb#10252]
## My Environment

* __ArangoDB Version__:        3.4.8
* __Storage Engine__:             RocksDB
* __Deployment Mode__:       Single Server
* __Deployment Strategy__:   Arangodb starter via Systemd
* __Configuration__:               N/A
* __Infrastructure__:               Regular cloud server
* __Operating System__:        Ubuntu 18.04
* __Total RAM in your machine__:        8Gb
* __Disks in use__:        SSD
* __Used Package__:              standalone

## Component, Query & Data

__Affected feature__:
Dashboard - statistics

## Steps to reproduce

1. Start arangodb server: sudo systemctl start arangodb
2. Open dashboard

__Problem__:
Statistics never change, even during import of data (e.g. restoring a previous dump with several thousands of records taking appr 1 minute to import).
Requests per second remains empty, transfer size per second remains empty, request types doesn't change, ... also the system resources remains the same.

__Expected result__:
bar charts and values should change/fluctuate over time, but nothing ever changes.

__systemd startup command__:
ExecStart=/opt/arangodb/bin/arangodb --starter.address=<hostname> --starter.port=<port> --starter.mode=single --starter.data-dir=/var/arangodb --auth.jwt-secret=<secret> --ssl.keyfile=<tls.keyfile> --all.server.maximal-threads=32

__remarks__:
- This is a clean install. The exact same startup command is used on another system with v3.4.5 where statistics are working fine.
- In Chrome I see the following error in the console after login:
```
VM16 app.js:1 Uncaught TypeError: Cannot read property 'el2' of undefined
    at VM16 app.js:1
    at Object.success (VM16 app.js:1)
    at j (VM15 libs.js:2)
    at Object.fireWith [as resolveWith] (VM15 libs.js:2)
    at x (VM15 libs.js:4)
    at XMLHttpRequest.<anonymous> (VM15 libs.js:4)
```

**Comment by santo74** [comment]
Update: I let it run for several hours and in the end statistics were suddenly available and constantly updated as expected. Not sure what triggered it.
Then I stopped the server at the end of the day and this morning I restarted it and again statistics don't work. Even the uptime stays at "a few seconds".

BTW, this is the correct systemd startup command:
`ExecStart=/opt/arangodb/bin/arangodb --starter.address=<hostname> --starter.port=<port> --starter.mode=single --starter.data-dir=/var/arangodb --auth.jwt-secret=<secret> --ssl.keyfile=<tls.keyfile> --all.server.maximal-threads=32 --all.foxx.api=false --all.foxx.store=false`
(Something went wrong with copy/paste in my previous post)

**Comment by santo74** [comment]
I think I found the cause of the issue.
My new cloud server was configured with a local timezone and also at the OS level I configured the timezone.
Apparently systemd always expects UTC (at least on Ubuntu 18.04) and because the server was configured with a different timezone it added 2 hours extra (in this case) to the actual time.
So when the server was started at e.g. 17:00 systemd thought it was 19:00 because of the configured timezone.
But at some point the timezone configured at the os level kicks in and the actual system time is being adjusted to the actual timezone, so in this example e.g. 19:02 suddenly became 17:02.
This confuses the systemd services and results in unexpected behaviour.

So I'm closing the issue because not setting a different timezone at the server level seems to solve it.
